healthtools/config.py

```diff
@@ -26,6 +27,7 @@
 
 TEST_DIR = os.getcwd() + "/healthtools/tests"
 
-SLACK = {
-    "url": os.getenv("WEBHOOK_URL")
-    }
+BATCH = 5
+HF_BATCH = 100  # batch for health facilities
+
+DATA_DIR = os.getcwd() + "/healthtools/"
```

Comment:
I notice now that you're saving it in the package.. (`healthtools/data`) I would advise against that and instead go for `data` only and have that as the `DATA_DIR`.